Reflection_Engine/Query/TypeList.cs

```diff
@@ -136,7 +137,7 @@ private static void ExtractAllTypes()
                     {
                         foreach (Type type in asm.GetTypes())
                         {
-                            if (type.Namespace != null && type.Namespace.StartsWith("BH.oM"))
+                            if (type.Namespace != null && type.Namespace.StartsWith("BH.oM") && typeof(IObject).IsAssignableFrom(type))
```

Comment:
This will prevent returning all BHoMAdapter types because they do not inherit from `IObject`. Is this intended?

Reply:
Disregard my comment above, I remembered that Adapters have a namespace that start with `BH.Adapter` so they were already excluded from this list.